apps/voting/contracts/Voting.sol

```diff
@@ -174,16 +192,6 @@ contract Voting is IForwarder, AragonApp {
         emit ChangeRepresentative(msg.sender, _representative);
     }
 
-    /**
-    * @notice Create a new vote about "`_metadata`"
-    * @param _executionScript EVM script to be executed on approval
-    * @param _metadata Vote metadata
-    * @return voteId Id for newly created vote
-    */
-    function newVote(bytes _executionScript, string _metadata) external auth(CREATE_VOTES_ROLE) returns (uint256 voteId) {
```

Comment:
Also wondering, are we able to repurpose the `string` metadata argument into the Agreement context?

Reply:
I think we are, that is indeed how the proposed implementation works